src/components/Annotation/index.js

```diff
@@ -1,17 +1,70 @@
 import React from 'react';
 import { annotationShape } from '../../utils/proptypes';
+import { boundedSeries } from '../../utils/boundedseries';
 
-const Annotation = ({ data, xScale, height, color, fillOpacity, id }) => (
-  <rect
-    key={id}
-    className={`griff-annotation griff-annotation-${id}`}
-    x={xScale(data[0])}
-    y={0}
-    height={height}
-    width={xScale(data[1]) - xScale(data[0])}
-    style={{ stroke: color, fill: color, fillOpacity }}
-  />
-);
+const Annotation = ({
+  timeBounds,
+  seriesBounds,
+  data,
+  xScale,
+  yScale,
+  height,
+  width,
+  color,
```

Comment:
color is an optional field; could we have it fall back to the color specified by the `seriesBounds.id` series?